models/stg_apple_search_ads__ad_group_report.sql

```diff
@@ -30,12 +30,15 @@ final as (
         source_relation,
         date as date_day, 
         ad_group_id,
-        conversions,
         impressions,
         local_spend_amount as spend,
         local_spend_currency as currency,
-        new_downloads,
-        redownloads,
+        coalesce(conversions, tap_installs) as conversions, 
+        coalesce(tap_installs, conversions) as tap_installs,
+        coalesce(new_downloads,tap_new_downloads) as new_downloads,
+        coalesce(tap_new_downloads,new_downloads) as tap_new_downloads,
+        coalesce(redownloads,tap_redownloads) as redownloads,
+        coalesce(tap_redownloads,redownloads) as tap_redownloads,
```

Comment:
```suggestion
        coalesce(new_downloads, tap_new_downloads) as new_downloads,
        coalesce(tap_new_downloads, new_downloads) as tap_new_downloads,
        coalesce(redownloads, tap_redownloads) as redownloads,
        coalesce(tap_redownloads, redownloads) as tap_redownloads,
```
Just to align on a more readable format